### DISCUSSION TITLE
About the home page display

### DISCUSSION THREAD

**Discussion by momo91377** [web-infra-dev/rspress#1392] [Q&A] [answered]
Whether the hero text specified in the home page is set to a default display length? Can I flexibly control it? Take the home page of Rspress as an example, can I control the boxed part of it without line breaks?
<img width="1005" alt="Snipaste_2024-09-05_17-00-38" src="https://github.com/user-attachments/assets/e6b6ee7d-6f3c-43d9-a073-17bf986cc9d0">

**Comment by chenjiahan** [comment, marked as answer]
You can override the `--rp-home-hero-text-max-width` CSS variable:

![image](https://github.com/user-attachments/assets/ec939b3f-2ffe-4efc-9be9-aaa8489f43f1)

**Reply by momo91377** [reply]
Get, I'll try.  Thanks a lot

**Reply by momo91377** [reply]
I try to specify it in css file, but it doesn't take effect. So whether this configuration item is only available in the latest version, my local version is 1.23.1, and there is no option to check this line with F12 mode.

**Reply by chenjiahan** [reply]
You can upgrade Rspress version first.

**Reply by momo91377** [reply]
Yes, I did that, but using the latest version 1.29.0 still the same result, this part of my project here is shown below:

![Uploading Snipaste_2024-09-05_18-06-26.png…]()

**Reply by chenjiahan** [reply]
The image is broken

**Reply by momo91377** [reply]
<img width="814" alt="Snipaste_2024-09-05_18-06-26" src="https://github.com/user-attachments/assets/d4ecd755-2303-4fda-b015-0a7daa8f6fd6">

**Reply by momo91377** [reply]
Sorry, I just reuploaded it.

**Reply by chenjiahan** [reply]
I'm not sure if you are using the latest version of Rspress, maybe you can provide a reproduction

**Reply by momo91377** [reply]
Sorry, the project is not yet officially available to the public, so I can't provide a reproducible project due to company confidentiality, I'll make a try on my own again, thanks a lot
